libs/feature/search/src/lib/utils/service/fields.service.ts

```diff
@@ -23,9 +23,23 @@ export class FieldsService {
       this.injector
     ),
     topic: new TopicSearchField(this.injector),
+    inspireKeyword: new SimpleSearchField(
+      'th_httpinspireeceuropaeutheme-theme_tree.default',
+      'asc',
+      this.injector
+    ),
+    documentStandard: new SimpleSearchField(
+      'documentStandard',
+      'asc',
+      this.injector
+    ),
     isSpatial: new IsSpatialSearchField(this.injector),
     q: new FullTextSearchField(),
-    license: new LicenseSearchField(this.injector),
+    license: new SimpleSearchField(
+      'licenseObject.default.keyword',
+      'asc',
+      this.injector
+    ),
```

Comment:
Same, this change shouldn't be part of this PR

Note that there might be an issue with your GN index configuration? We should look into this seperately

Reply:
@jahow with this in mind we should probably drop the PR entirely, because the added 2 filters are working in a similar manner and so might break other things. 

What do you think?

Reply:
Well the two added filters have been working great in my tests, so I would definitely be in favor or taking them in. Adding new filters will not break anything, but modifying existing ones might.

Reply:
@jahow fair enough. I removed any changes to the existing filters from the PR.